run-tests.sh

```diff
@@ -93,6 +90,6 @@ RunTests() {
 
 # Will not run if sourced for bats-core tests.
 # View src/tests for more information.
-if [ $BITRISE_TEST_ENV != true ]; then
+if [[ $BITRISE_TEST_ENV != true ]]; then
```

Comment:
Was is failing without the double square brackets?

Reply:
Even worse it was giving me a warning didn't execute anything but the command was marked  as successful :D

Reply:
It's because `true` isn't a string. And all environment variables are strings, so you are doing `"true" === true` basically.

I would prefer staying explicit that we are working with string variables, and go back to something like we did in CircleCI:

```
if [ "${0#*$ORB_TEST_ENV}" == "$0" ]; then
```

So the following should work:
```suggestion
if [ "$BITRISE_TEST_ENV" != "true" ]; then
```

Reply:
Also, let's not call `BITRISE_TEST_ENV` this way, because it sounds like it's a native Bitrise env var.

Something like `IS_TEST_ENV` would be better 😁